modules/ROOT/pages/proc_defining-and-creating-pipelineresources.adoc

```diff
@@ -5,14 +5,14 @@
 [id="defining-and-creating-pipelineresources_{context}"]
 = Defining and creating pipeline resources
 
-`PipelineResources` are artifacts that are used as inputs to a `Task` and can be output by a `Task`.
+Pipeline resources are artifacts that are used as inputs to a task and can be output by a task.
```

Comment:
Any reason for the change?

Reply:
We use back-ticks to either refer to a command or for file names or directories. In this case it is an introductory sentence for the section. So we are removing the back ticks as we go. We will do a full sweep of such occurrences and remove them from the docs to make it consistent.

Also, `task` s looks very awkward